tests/scipy_signal_test.py

```diff
@@ -91,6 +91,34 @@ def testConvolutions(self, xshape, yshape, dtype, mode, op):
     self._CheckAgainstNumpy(osp_fun, jsp_fun, args_maker, check_dtypes=False, tol=tol)
     self._CompileAndCheck(jsp_fun, args_maker, rtol=tol, atol=tol)
 
+  # Test  fftconvolve: present JAX code implement only mode="same"
+  @jtu.sample_product(
+    dtype=default_dtypes,
+    shape=[
+      # x_input_shape, y_input_shape=x_input_shape
+      ((1,), (1,)),
+      ((1,1), (1,1)), 
+      ((5,5), (5,5)),
+      ((5,5), (2,2)),
+      ((10,10),(3,3)),
+      ((10,10), (10,10)),
+      ((3,3),(10,10)),
+      ((10,5),(3,2)),
+    ],
```

Comment:
You could use pre-defined shapesets for this, similar to https://github.com/google/jax/blob/6f2256ad17304ab2be667cdfc49302125220b81c/tests/scipy_signal_test.py#L74-L78

Reply:
ok. done. I though my tests was a little more expressive as the code can accomodate more dimensions.